### PR TITLE
Add order completed event

### PR DESCRIPTION
**Pull request by cgebken**

### CODE REVIEW

File: apps/change-log.md [hunk 1]
```diff
@@ -14,12 +14,14 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
-## 2019-07-18
+## 2019-07-26
 
 ### Changes with software release 7.37.0
```
Comment (cgebken) [inline apps/change-log.md]: @depoulo It is available in that version is it? Or will it only be available in the 7.38.0?

Comment (depoulo) [reply apps/change-log.md]: Well it's in epages-ui, [versuon 2.5.0](https://github.com/ePages-de/epages-ui/releases/tag/v2.5.0).

File: apps/events.md [hunk 2]
```diff
@@ -152,20 +153,48 @@ checkoutState: null
...
+subAmount: "76,01 €"
 taxType: "NET"
 taxes: []
 totalNumberOfItems: 3
 _links: null
 ```
 
+## Order:completed event
+
+In order to make use of this event, you need to add the following snippet to your code:
```
Comment (BirgitBader) [inline apps/events.md]: To make use of this event...

File: apps/events.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ These events are available:
 | `product` | immutable.js object | Is triggered when the customer selects a product. Informs about the respective product. |
 | `category` | immutable.js object | Is triggered when the customer selects a category. Informs about the respective category and related products.|
 | `cart:add` | object | Is triggered when the customer adds a product to the cart. Informs about the current state of the cart, e.g. included items. This event isn't triggered if the customer makes changes in the cart itself, e.g. changing the amount of an item. |
+| `order:completed` | object | Is triggered when the customer reaches the order confirmation page after completing a purchase. Gives further information on the order, e.g. on the billing address, the order number, and the selected shipping method. |
```
Comment (BirgitBader) [inline apps/events.md]: Gives further information on the order, e.g. > Provides additional information about the order, such as

**Pull request merged**